README.md

```diff
@@ -75,11 +78,12 @@ to the filesystem using normal calls:
     }
 
 ### Building Linux
-To build a SMP Linux executable for BlackParrot:
+To build a SMP Linux executable for BlackParrot (make sure first to follow the above instructions for building the SDK):
 ```
 make -j linux OPENSBI_NCPUS=<n>
+./install/bin/dromajo --host --ncpus=8 linux/linux.riscv # verify linux runs on the dromajo simulator
```

Comment:
`./install/bin/dromajo --host --ncpus=<n> prog/linux/linux.riscv`

`linux/linux.riscv` is also correct, but we should probably refer to the prog directory here.

Reply:
I think linux/linux.riscv is better because they might be building in that directory locally for whatever reason

Reply:
I don't feel strongly about this either way.
Have to fix the `--ncpus` thing before merging though.

Reply:
fixed